src/testdrive/src/action.rs

```diff
@@ -472,11 +573,13 @@ pub async fn create_state(
         kafka_config,
         kafka_producer,
         kafka_topics,
-        aws_region: config.aws_region.clone(),
+        aws_region: aws_info.region,
         aws_account: config.aws_account.clone(),
-        aws_credentials: config.aws_credentials.clone(),
+        aws_credentials: aws_info.credentials.unwrap().into(),
```

Comment:
What's up with this `unwrap`?

Reply:
Similar to above, changed it to an expect with explanation.